code-studio/scripts/build-commands.js

```diff
@@ -295,3 +337,25 @@ function warnIfWrongNodeVersion() {
       console.log('You are using node ' + process.version + '. This build script expects ' + engines_node + '.');
   }
 }
+
+/**
+ * Wraps a function so that calling it with a set of arguments returns a new
+ * function with those arguments pre-bound - essentially a one-time curry.
+ * @param {!function} fn
+ * @returns {Function}
+ */
+function makeBuildCommand(fn) {
+  return function () {
+    return fn.bind.apply(fn, [undefined].concat([].slice.call(arguments)));
+  };
+}
+
+// Export wrapped versions of the commands we need, suitable for passing
+// into Promise.prototype.then()
+_.extend(module.exports, {
+  copyDirectory: makeBuildCommand(copyDirectory),
+  ensureDirectoryExists: makeBuildCommand(ensureDirectoryExists),
+  executeShellCommandsInParallel: makeBuildCommand(executeShellCommandsInParallel),
+  logSuccess: makeBuildCommand(logSuccess),
+  logFailure: makeBuildCommand(logFailure)
+});
```

Comment:
Any reason we're extending module.exports here instead of just setting it? Doesn't look like we should have anything in module.exports currently...